profile/tests/test_views.py

```diff
@@ -147,7 +147,10 @@ def test_request_reset_password_email_wrong_email(self):
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
 
     def test_get_reset_page_ok(self):
-        url = reverse('account_reset_password_from_key', kwargs={'uidb64': self.uid, 'token': self.token})
+        try:
+            url = reverse('account_reset_password_from_key', kwargs={'uidb64': self.uid, 'token': self.token})
+        except Exception as e:
```

Comment:
Why are you excepting with `Exception`?, this is usually a bad practice.
Also, regulating the flow with `try:` `except:` is also usually a bad practice.